qutebrowser/misc/utilcmds.py

```diff
@@ -234,3 +234,17 @@ def repeat_command(win_id, count=None):
     cmd = runners.last_command[mode_manager.mode]
     commandrunner = runners.CommandRunner(win_id)
     commandrunner.run(cmd[0], count if count is not None else cmd[1])
+
+@cmdutils.register(debug=True,name='debug-log-capacity')
```

Comment:
Please add a space after the comma here (generally, with arguments there's always a space after commas).